### ISSUE TITLE
Menu bar at wrong position on Mac OS since commit bec55f8

### ISSUE THREAD

**Issue by amaxi99** [JabRef/jabref#10966]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

macOS

### Details on version and operating system

macOS 14.3

### Checked with the latest development build (copy version output from About dialog)

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

Since commit [bec55f8](https://github.com/JabRef/jabref/commit/bec55f8c42430221838fbd18a91fe652e2408dda) the menu bar is included into the main window instead of the Mac OS menu bar (as before this commit).

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
Yeah, I could verify this as well here, needs investigation what exactly happened
